snakemd/elements.py

```diff
@@ -354,24 +340,27 @@ class Paragraph(Block):
     A paragraph is a standalone block of text. Paragraphs can be
     formatted in a variety of ways including as blockquotes.
 
-    :param Iterable[Inline | str] content: a "list" of text objects to render as a paragraph        
+    :param str | Iterable[Inline | str] content: a "list" of text objects to render as a paragraph        
     :param bool quote: the quote state of the paragraph;
         set True to convert the paragraph to a blockquote (i.e., True -> > quote)
     """
 
-    def __init__(self, content: Iterable[Inline | str], quote: bool = False):
+    def __init__(self, content: str | Iterable[Inline | str], quote: bool = False):
```

Comment:
The `|` only works in python 3.10 and above. If you need to support pre-3.10, you need to use `Union` instead. For something this complex, you might be better off defining a type like this:
```python
ContentT = Union[str, Iterable[Union[Inline, str]]]
```